compiler/qsc_ast/src/ast.rs

```diff
@@ -262,6 +272,8 @@ pub enum ItemKind {
     Open(Idents, Option<Box<Ident>>),
     /// A `newtype` declaration.
     Ty(Box<Ident>, Box<TyDef>),
+    /// An export declaration
+    Export(ExportDecl),
```

Comment:
How do we handle item properties on export and import items?

```qsharp
/// doc comment!
@Config(Base)
internal import Bar.BarOperation;
```

I guess the current answer is that they don't cause any errors and have no effect, but is this desirable?